lib/lumberjack/server.rb

```diff
@@ -87,17 +98,17 @@ def transition(state, next_length)
     end # def transition
 
     # Feed data to this parser.
-    # 
+    #
     # Currently, it will return the raw payload of websocket messages.
     # Otherwise, it returns nil if no complete message has yet been consumed.
     #
-    # @param [String] the string data to feed into the parser. 
+    # @param [String] the string data to feed into the parser.
     # @return [String, nil] the websocket message payload, if any, nil otherwise.
     def feed(data, &block)
       @buffer << data
       #p :need => @need
       while have?(@need)
-        send(@state, &block) 
```

Comment:
Can you fix this PR to not include unintentional whitespace changes? It makes it harder to review and upsets `git blame` in the future :)